### PR TITLE
Add definition for barbarian

### PR DESCRIPTION
**Pull request by codeAbul**
- Adds definition for barbarian with details of usage, issues and sources. 

( sidenote: My first ever OSS pull request. Thanks to @tatianamac for encouraging my submission. Was always intimidated about contributing to OSS )

### CODE REVIEW

File: 11ty/definitions/barbarian.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Barbarian
```
Comment (tatianamac) [inline 11ty/definitions/barbarian.md]: This is pedantic, but 'barbarian' is a noun, and all the alt-words are adjectives. 

One solution is to make it barbaric instead.

File: 11ty/definitions/barbarian.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  level: 'Avoid'
+defined: true
+speech: noun
+alt_words:
+  - cruel
+  - vicious
+  - obscene
+  - feral
+---
+someone who is "savagely cruel"; primitive ; unsophisticated
```
Comment (tatianamac) [inline 11ty/definitions/barbarian.md]: I imagine this is from a dictionary but we should avoid "savagely" here as I would want to offer that connection as problematic. (I imagine you recognised this given it's in quotes). 

Also remove space after primitive and before semicolon.

File: 11ty/definitions/barbarian.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+
+#### Issues
+
+Barbarian originates from the greek word *bárbaros* meaning "babbler", to denote the "unintelligle sounds" (_"bar bar bar") made by foreign speakers.
```
Comment (tatianamac) [inline 11ty/definitions/barbarian.md]: Greek should be capitalised

File: 11ty/definitions/barbarian.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+Similar words exist in many other languages, for the identical purpose labeling a "strange"/"foriegn" person/culture.
```
Comment (tatianamac) [inline 11ty/definitions/barbarian.md]: foreign*

File: 11ty/definitions/barbarian.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+someone who is "savagely cruel"; primitive ; unsophisticated
+
+
+#### Issues
+
+Barbarian originates from the greek word *bárbaros* meaning "babbler", to denote the "unintelligle sounds" (_"bar bar bar") made by foreign speakers.
+Similar words exist in many other languages, for the identical purpose labeling a "strange"/"foriegn" person/culture. 
+
+In modern day usage, you can notice this rhetoric being employed almost always for the purpose of demonizing "foreign" (more often than not from the global south) cultures - cultures that need to be civilized;
+cultures that need to be colonized to rescue them from themselves. A perfect lingual weapon for warmongers and racists alike. Equally repulsive sibling of "savage".
```
Comment (tatianamac) [inline 11ty/definitions/barbarian.md]: Let's break out "savage" as a "Related Words" section and link it.

cc: @ovlb Is there anything we should consider in doing this? I'd like for those words to eventually have some way of connecting.

File: 11ty/definitions/barbarian.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+Barbarian originates from the greek word *bárbaros* meaning "babbler", to denote the "unintelligle sounds" (_"bar bar bar") made by foreign speakers.
+Similar words exist in many other languages, for the identical purpose labeling a "strange"/"foriegn" person/culture. 
+
+In modern day usage, you can notice this rhetoric being employed almost always for the purpose of demonizing "foreign" (more often than not from the global south) cultures - cultures that need to be civilized;
+cultures that need to be colonized to rescue them from themselves. A perfect lingual weapon for warmongers and racists alike. Equally repulsive sibling of "savage". 
+
+#### Impact
+
+When you use words, that are in vogue primarily inside circles of bigots, with racist etymologies, it's a wilful act of violence, ignorance and an indication to the vilified and minoritized communities that their history of wounds and abuses are inconsequential to our words.
```
Comment (tatianamac) [inline 11ty/definitions/barbarian.md]: We use International English so it should be minoritised.

**Comment by tatianamac** [comment]
Thank you so much for contributing, @codeAbul ! I added a few editorial notes for you.

@ovlb : Do you mind taking a look at the syntax to ensure it's good with you?

File: 11ty/definitions/barbarian.md (hunk 5 above)
Comment (ovlb) [reply 11ty/definitions/barbarian.md]: The one-way connection/link in the alternative words (e.g. barbarian -> savage) is automatic already. We can also add a «words that link here» section in the build. So I think there’s nothing special to consider currently. I will open an issue to discuss this further :)

**Comment by ovlb** [comment]
The strcuture looks fine :)

**Comment by codeAbul** [comment]
@tatianamac Thanks a lot for the suggestions. Will do it!

**Comment by codeAbul** [comment]
@tatianamac Please check when you have time. Sorry for the late response!

**Comment by tatianamac** [comment]
👏🏽 Congrats and thank you, @codeAbul, for your first contribution here! We appreciate it.

**Pull request merged**